keras_cv/ops/within_box_3d_test.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2022 The KerasCV Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import os
+from datetime import datetime
+
+import numpy as np
+import pytest
+import tensorflow as tf
+
+import keras_cv
+
+num_points = 200000
+num_boxes = 1000
+box_dimension = 20.0
+
+
+def get_points_boxes():
```

Comment:
Do you ensure no overlap between two bounding boxes? The v2 op assumes no overlaps, right?

Reply:
Good point. Added docstring